pagopa/io-app
# chore: [IOPID-654] Change insert new email screen

## Short description
To the present, we have not defined the names of the new data that should arrive at BE, so there are TODOs in the code. 
If the email does not comply with the CDU (email uniqueness criteria) then the user will see an information banner explaining that the email is already in use and if they try to use that email they will get the red error "email is already in use" and will not be able to proceed (the button is disabled). The user also has no possibility to proceed if the format is invalid.
If they proceed and the email was already in use, an information alert would be issued which would take the user back to the input page.

## List of changes proposed in this pull request
- Create new page 
- Unit test on the new page
- Add tanslations

## How to test
Into the io-dev-server-api set firstOnboarding field as true (into config.ts file), run `yarn generate` and then uninstall the io app from the device (or emulator) and then run the app and the server. Into the io-app, in OnboardingNavigator.tsx file change line 59 from OnboardingEmailInsertScreen to NewOnboardingEmailInsertScreen. 
if you want to do the first onboarding as if the email respect the CDU you can set the state isCduEmail as true (at line 67) and if you want to see the alert at the click you can set the showAlertExistsEmail const as true.
